.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,4 +23,4 @@ Feel free to ping committers for the review!
 ## Check list
 
 * [ ] Code changed are covered with tests, or it does not need tests for reason:
-* [ ] Change does not need document change, or I will submit document change to https://github.com/apache/incubator-seatunnel-website later
+* [ ] Update the documentation to describe this new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
```

Comment:
I prefer to use keyword `if this pull request add new feature` or something like that

Reply:
how about `If necessary, please update the documentation to describe the new features.`

Reply:
thx,done~